src/ServiceControl/Monitoring/Handler/NewEndpointDetectedHandler.cs

```diff
@@ -13,8 +13,7 @@ public NewEndpointDetectedHandler(EndpointInstanceMonitoring endpointInstanceMon
 
         public Task Handle(NewEndpointDetected message, IMessageHandlerContext context)
         {
-            endpointInstanceMonitoring.DetectEndpointFromRemoteAudit(message.Endpoint);
-            return Task.FromResult(0);
+            return endpointInstanceMonitoring.EndpointDetected(message.Endpoint);
```

Comment:
Not sure if this will actually persist the endpoint, will add some tests for this